TOOLS/ce/src/main/scala/org.opalj.ce/FileLocator.scala

```diff
@@ -0,0 +1,108 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.nio.file.attribute.BasicFileAttributes
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+import scala.jdk.CollectionConverters._
+import com.typesafe.config.Config
+
+/**
+ * File Locator aids locating the config Files that the configuration Explorer needs to parse
+ * @param config accepts the config of the Configuration Explorer
+ */
+class FileLocator(var config: Config) {
```

Comment:
`ConfigFileLocator`?

Reply:
Not really. It is not limited to finding config files.
It can search a file tree for all possible project files.
In fact, it is used for both finding config files and jar archives in this project.

Reply:
Then the ScalaDoc above is incorrect

Reply:
It is outdated indeed

Reply:
I fixed it now